app/root.tsx

```diff
@@ -232,16 +184,63 @@ export default function App() {
             <>
               <Atlas />{" "}
               <Overlay>
-                <Show above="lg">
+                <Flex
+                  direction={"column"}
+                  width={{ base: "100%", lg: "auto" }}
+                  alignItems={"center"}
+                  flexShrink={{ lg: 0 }}
+                  maxHeight={{ lg: "100%" }}
+                  overflowY={{ lg: "auto" }}
```

Comment:
Adding `overflowX={{ lg: "hidden" }}` here should fix the minor issue of getting a horizontal scroll bar on short desktop screen sizes.

Reply:
change pushed. I'll leave it for you to resolve when the behavior is confirmed gone.